### ISSUE TITLE
BES 0100: full specification of block "Unknown 0x036" (0x0036)

### ISSUE THREAD

**Issue by sonicpp** [OpenVietcong/vc-spec#9]
Full documentation and script checking of "Unknown 0x036" block for BES file format of version 100

**Comment by sonicpp** [comment]
It looks like it is some kind of link/copy to other models (can by across different object parents). Model which contains mesh sub-block and this 0x36 is a reference model (which has unique id in this sub-block), other models containing this sub-block with the same id are just the copies of the reference model and thus can not contain any mesh sub-block.

**Comment by FlashX64** [comment]
Yep. It's used for mesh instancing on the model file level The first occurance of this block within some mesh object registers a unique numeric identifier. Other meshes then may reference its mesh data using the same identifier and thus save space in the file.
